src/airflow/dags/common_airflow.py

```diff
@@ -106,7 +106,9 @@ def create_cluster(
             # Create a disk config section if it does not exist.
             cluster_config[worker_section].setdefault("disk_config", dict())
             # Specify the number of local SSDs.
-            cluster_config[worker_section]["num_local_ssds"] = num_local_ssds
+            cluster_config[worker_section]["disk_config"][
```

Comment:
How did this work before?

Reply:
This did work at first; then I was rebasing a previous PR to modify the list of changes included with it and accidentally broke it; and now after the fix it works again